src/internal/modules/charge-information/forms/start-date.js

```diff
@@ -44,13 +44,21 @@ const getDates = licence => {
   };
 };
 
-const minErrors = {
-  [MIN_LICENCE_START]: 'You must enter a date after the licence start date'
+const getDatesWithDocumentRoles = (licence, licenceDocumentsRoles) => {
+  const startDate = moment(licenceDocumentsRoles.startDate);
+  const minDate = startDate;
+  const isLicenceStart = startDate.isAfter(minDate);
```

Comment:
Thi block of code here doesn't make sense. If startDate = 1 Jan 2001 and minDate = startDate then startDate will never be after minDate because minDate is also 1 Jan 2001.